core/src/test/java/nl/nn/adapterframework/jdbc/FixedQuerySenderTest.java

```diff
@@ -7,7 +7,7 @@
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.junit.MatcherAssume.assumeThat;
+import static org.junit.Assume.assumeThat;
```

Comment:
For Junit 5 tests you need to import `org.junit.jupiter.api.Assumptions` instead.

Reply:
Changed them to assumeTrue for Junit 5